rye/src/cli/version.rs

```diff
@@ -25,8 +28,37 @@ pub fn execute(cmd: Args) -> Result<(), Error> {
             eprintln!("version set to {}", version);
         }
         None => {
-            eprintln!("{}", pyproject_toml.version()?);
+            let mut version = pyproject_toml.version()?;
+            match cmd.bump {
+                Some(bump) => bump_version(&mut version, &bump, &mut pyproject_toml),
+                None => eprintln!("{}", version),
+            }
         }
     }
     Ok(())
 }
+
+fn bump_version(version: &mut Version, bump: &str, pyproject: &mut PyProject) {
+    if version.is_post() {
+        version.post = None;
+    }
+    if version.is_dev() {
+        version.dev = None;
+        eprintln!(
+            "{} dev version will be bumped to release version",
+            style("warning:").red()
+        );
+    } else {
+        match bump {
+            "major" => version.release[0] += 1,
+            "minor" => version.release[1] += 1,
+            "patch" => version.release[2] += 1,
+            _ => return,
```

Comment:
It would be good to give an error here on invalid input. Right now i think you could use an enum and then clap would handle it.

Reply:
Thanks for the suggestion, enum is indeed in my consideration.